corehq/apps/translations/app_translations/upload_app.py

```diff
@@ -96,6 +109,38 @@ def process_bulk_app_translation_upload(app, workbook, expected_headers, lang=No
     return msgs
 
 
+def get_sheet_name_to_unique_id_map(file_or_filename, lang):
+
+    def get_sheet_name():
+        return MODULES_AND_FORMS_SHEET_NAME if is_multisheet() else SINGLE_SHEET_NAME
+
+    def is_multisheet():
+        return lang is None
+
+    def is_modules_and_forms_row(row):
+        # Rows about the app's modules and forms, like their names and unique IDs
+        return not row['case_property'] and not row['list_or_detail'] and not row['label']
```

Comment:
Can we just check if there is a value in column 'unique_id' or is that always populated?

nit: this might read better as
```
def is_not_module_or_form_row(row):
     return not( row['case_property'] or row['label'])
```

I think checking for `case_property` should be enough for module.

Reply:
I think just checking `row['unique_id']` should definitely work, and is much simpler.